lib/common/options.js

```diff
@@ -64,18 +64,16 @@ Options.from = function (_options) {
   // Populate modulesDir
   options.modulesDir = []
     .concat(options.modulesDir)
-    .concat(path.join(options.nuxtDir, 'node_modules'))
-    .filter(hasValue)
+    .concat(path.join(options.nuxtDir, 'node_modules')).filter(hasValue)
     .map(dir => path.resolve(options.rootDir, dir))
 
-  // Sanitize extensions
-  if (!options.extensions.includes('js')) {
-    options.extensions.unshift('js')
-  }
+  const extensionsToRemoveIfPresent = ['js', 'mjs']
 
-  if (!options.extensions.includes('mjs')) {
-    options.extensions.unshift('mjs')
-  }
+  extensionsToRemoveIfPresent.forEach((ext) => {
```

Comment:
This is an extra function invocation for nothing. Why is this worse than `for of`?

Reply:
It's a matter of style/opinions again as both should actually have (nearly) the same runtime (esp. for **2 elements**). I prefer the functional/declarative style.

Reply:
tbh, I must agree with that.
But imho in this particular case what is taking place conceptually is a filtering of the static array into missing values, which are then  added  to the front of the array. Why the front and not the back, I am not 100% sure.